android/src/main/java/com/tailscale/ipn/ui/util/LoadingIndicator.kt

```diff
@@ -0,0 +1,52 @@
+package com.tailscale.ipn.ui.util
+
+import android.util.Log
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import kotlinx.coroutines.flow.MutableStateFlow
+
+object LoadingIndicator {
```

Comment:
Some operations like setting the exit node can be quite slow, so having a loading indicator seems appropriate.